From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Mon, 1 Jul 2024 20:27:30 -0600
Subject: [PATCH] kernelbase: HACK: Force disable direct composition for Super.

CW-Bug-Id: #23959
Valve-commit: d64a2281bb3e9d581d7c3324e7136f1d16a4b65d
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-wine-proton-rebase.md
Offered-upstream: no
---
 dlls/kernelbase/process.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/kernelbase/process.c b/dlls/kernelbase/process.c
index 0d4f888..a9c398b 100644
--- a/dlls/kernelbase/process.c
+++ b/dlls/kernelbase/process.c
@@ -511,6 +511,7 @@ static const WCHAR *hack_append_command_line( const WCHAR *cmd )
     }
     options[] =
     {
+        {L"Super\\Super.exe", L" --disable_direct_composition=1"},
         {L"A Raven Monologue.exe", L" --use-angle=d3d9"},
         {L"antenna\\antenna.exe", L" --use-angle=d3d9"},
         {L"Bloody Walls\\game.exe", L" --disable_direct_composition=1"},
